src/ui/public/stringify/types/String.js

```diff
@@ -55,10 +57,15 @@ export default function _StringProvider(Private) {
     }
   };
 
+  _String.prototype._toTitleCase = function (val) {
+    return val.replace(/\w\S*/g, txt => { return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase(); });
```

Comment:
@patrickfournier That's where I saw it and thought it should probably be the same. Good point...